### PR TITLE
[LocalB] New localbranch

### PR DESCRIPTION
**Pull request by kulkvivek**
This is to validate the changes to localbranch

### CODE REVIEW

**Comment by vivekkulkarni-ops** [comment]
> This is to validate the changes to localbranch

No need it's already done. This will be done at a later stage in life.

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# first-github-repo
\ No newline at end of file
+# first-github-repo
+nEW LOCALBranch made.
```
Comment (kulkvivek) [inline README.md:2]: What about this line of code.

Comment (kulkvivek) [reply README.md:2]: How do we take this forward?

**Comment by vivekkulkarni-ops** [comment]
Sorry

**Comment by vivekkulkarni-ops** [comment]
Sorry but I dont think this needs to be merged at this moment of time.

**Comment by vivekkulkarni-ops** [comment]
See you nect time buddy

**Comment by kulkvivek** [comment]
Okay brother

**Pull request closed without merging**